contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -289,6 +308,41 @@ contract VestingRegistryLogic is VestingRegistryStorage {
         return vestings[uid].vestingAddress;
     }
 
+    /**
+     * @dev check if the specific vesting address is team vesting or not
+     * @dev read the vestingType from vestingDetail storage
+     *
+     * @param _vestingAddress address of vesting contract
+     *
+     * @return true for teamVesting, false for normal vesting
+     */
+    function isTeamVesting(address _vestingAddress) external view returns (bool) {
+        if (
+            vestingDetail[_vestingAddress].vestingAddress == address(0) ||
+            vestingDetail[_vestingAddress].vestingType != uint256(VestingType.TeamVesting)
+        ) return false;
+        else return true;
+    }
+
+    /**
+     * @dev setter function to register existing vesting contract to vestingDetail storage
+     * @dev need to set the function visilibty to public to support Vesting struct as parameter
+     *
+     * @param vestings array for Vesting struct
+     */
+    function registerVestingToVestingDetail(Vesting[] memory vestings) public onlyAuthorized {
+        for (uint256 i = 0; i < vestings.length; i++) {
+            Vesting memory _vesting = vestings[i];
+            require(_vesting.vestingAddress != address(0), "ZERO_VESTING_ADDRESS");
+            vestingDetail[_vesting.vestingAddress] = _vesting;
```

Comment:
```suggestion
            vestingDetails[_vesting.vestingAddress] = _vesting;
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)